e2e/test/iothub/MqttTransportHandlerShutdownTests.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Pay attention to the timeout and the "close to" amounts. The idea was to give enough wiggle room but take into account of a busy thread scheduler.

These values produced the most consistent results with my testing.